version.py

```diff
@@ -0,0 +1,26 @@
+import os
+import toml
+
+class VersionUtil:
+
+    @staticmethod
+    def find_toml_file():
+        for file in os.listdir('.'):
+            if file.endswith('.toml'):
+                return file
```

Comment:
You can blob until you run into a pyproject.toml file, so we can find the root of the project. We assume that any package that uses libversion has pyproject.toml at it's root.

Maybe it's worth mentioning the assumption in the readme.md?